### ISSUE TITLE
Cannot import d3-geo in node with typescript 

### ISSUE THREAD

**Issue by sardounis** [d3/d3-geo#261]
It seems I have an issue of importing the library using nodejs 14 and typescript.

I have used @types/d3-geo as well.

Though I AM GETTING THE ERROR: `An unexpected error occurred Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/antonissardounis/Projects/beat/bhs-cf/functions_new/functions/node_modules/d3-geo/src/index.js\nrequire() of ES modules is not supported.\nrequire() of /Users/antonissardounis/Projects/beat/bhs-cf/functions_new/functions/node_modules/d3-geo/src/index.js from /Users/antonissardounis/Projects/beat/bhs-cf/functions_new/functions/lib/utils/Geo/Geo.js is an ES module file as it is a .js file whose nearest parent package.json contains \"type\": \"module\" which defines all .js files in that package scope as ES modules.\nInstead rename index.js to end in .cjs, change the requiring code to use import(), or remove \"type\": \"module\" from /Users/antonissardounis/Projects/beat/bhs-cf/functions_new/functions/node_modules/d3-geo/package.json.\n\n    at new NodeError (internal/errors.js:322:7)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1102:13)\n    at Module.load (internal/modules/cjs/loader.js:950:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:790:12)\n    at Module.require (internal/modules/cjs/loader.js:974:19)\n    at require (internal/modules/cjs/helpers.js:101:18)\n    at /Users/antonissardounis/Projects/beat/bhs-cf/functions_new/functions/lib/utils/Geo/Geo.js:33:75\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)\n    at async Geo.init (/Users/antonissardounis/Projects/beat/bhs-cf/functions_new/functions/lib/utils/Geo/Geo.js:33:27)\n    at async Geo.AreCoordinatesInsideGeofence (/Users/antonissardounis/Projects/beat/bhs-cf/functions_new/functions/lib/utils/Geo/Geo.js:39:9) {\n  code: 'ERR_REQUIRE_ESM'\n}"}`

it seems there is an open issue at stack overflow as well: [link](url)

**Comment by Fullstack-WEB-Blockchain-Developer** [comment]
@sardounis Did you solve this issue?

**Comment by Toolenaar** [comment]
downgrading to v2.0.1 worked for me

**Comment by razzeee** [comment]
More people with this problem here https://github.com/yanivam/react-svg-worldmap/issues/135

**Comment by bennodev19** [comment]
any updates on this?

**Comment by bennodev19** [comment]
ok my mistake.. I didn't import it as namespace.

```ts
// index.ts
import * as d3Geo from 'd3-geo'

// compiled index.js
const d3Geo = require('d3-geo')
```

**Comment by bennodev19** [comment]
ok the esmodule import only works with `2.0.2` but not with the latest one `3.1.0` :/

**Comment by Fil** [comment]
this works for me:
> d3Geo = await import("d3-geo");

**Comment by bennodev19** [comment]
yeah but dynamic imports aren't that nice.. so I'll stick to the 2.x.x for now :)
